include/boost/url/impl/url.ipp

```diff
@@ -354,6 +354,13 @@ set_scheme(string_view s)
     return *this;
 }
 
+url&
+url::
+set_scheme(pct_encoded_view s)
```

Comment:
Umm... schemes are never encoded. This should be `string_view` only.

Reply:
`s` dereferences to decoded characters

Reply:
it doesn't matter, `pct_encoded_view` makes no sense here

Reply:
Yes. I've already removed it.